RLTSquare/Ccq/etc/queue_publisher.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework-message-queue:etc/publisher.xsd">
+    <publisher topic="rltsquare_hello_world.topic">
+        <connection name="amqp" exchange="rltsquare_hello_world.exchange" />
```

Comment:
Overcomplicated code - there is no need to declare connection here, the default will be picked up.